File: Python-solutions/2721.py
from itertools import cycle

snowballs = input()

reindeers = ["Dasher", "Dancer", "Prancer", "Vixen",
             "Comet", "Cupid", "Donner", "Blitzen",  "Rudolph"]


def parseNumSnowballs(entrada):
    total = 0
    entrada = entrada.split(" ")
    for i in entrada:
        total += int(i)

    return total


def pickReindeer(reindeers, num_snowballs):
    reindeers = cycle(reindeers)

    for r in reindeers:
        num_snowballs -= 1

        if(num_snowballs == 0):
            return r


num_snowballs = parseNumSnowballs(snowballs)
winner = pickReindeer(reindeers, num_snowballs)

print(winner)

File: C-solutions/2721.c
#include <stdlib.h>
#include "stdio.h"
#define size 9

int main(void) {
  int i;
  int ballssum, balls[size];
  
  for(i=0; i<size; i++){
    scanf(" %d", &balls[i]);
  }
  
  ballssum = 0;
  
  for(i=0; i<size; i++)
    ballssum+=balls[i];
  
  switch((ballssum%9)){
      case 1: printf("Dasher\n");break;
      case 2: printf("Dancer\n");break;
      case 3: printf("Prancer\n");break;
      case 4: printf("Vixen\n");break;
      case 5: printf("Comet\n"); break;
      case 6: printf("Cupid\n");break;
      case 7: printf("Donner\n");break;
      case 8: printf("Blitzen\n");break;
      case 0: printf("Rudolph\n");break;
      default: break;
  }
  
  return 0;
}
